sdks/python/setup.py

```diff
@@ -504,6 +501,19 @@ def get_portability_package_data():
               # https://github.com/apache/beam/issues/31285
               # 'xgboost<2.0',  # https://github.com/apache/beam/issues/31252
           ],
+          'p312_ml_test': [
+              'datatable',
+              'embeddings',
+              'onnxruntime',
+              'sentence-transformers',
+              'skl2onnx',
+              'pillow',
+              'tensorflow',
+              'tensorflow-hub',
```

Comment:
Removing `tensorflow-transform'` since Python 3.12 does not work with it now.

Reply:
Since py39-311 already test MLTransform so it is safe to ignore this for Py312 for now.